### PR TITLE
[libdef] platform v1.x

### PR DESCRIPTION
**Pull request by bassettsj**
- Add libdef for [platform](https://github.com/bestiejs/platform.js/blob/master/doc/README.md#readme)
- Add tests for it.

### CODE REVIEW

File: definitions/npm/platform_v1.x.x/flow_>=v0.21.x/platform_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+declare module 'platform' {
+  declare class Platform extends Object {
```
Comment (bassettsj) [inline definitions/npm/platform_v1.x.x/flow_>=v0.21.x/platform_v1.x.x.js]: It is supposed to be an object, but defining the class makes more sense for the parse method which returns a new instance.

File: definitions/npm/platform_v1.x.x/test_platform_v1.x.x.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// @flow
+import platform from 'platform';
+import type { $Platform } from 'platform';
```
Comment (marudor) [inline definitions/npm/platform_v1.x.x/test_platform_v1.x.x.js]: Why the extra type import/export?
you can just use platform as type.

Comment (bassettsj) [inline definitions/npm/platform_v1.x.x/test_platform_v1.x.x.js]: Yup, seems good. `typeof platform` works just fine.

**Comment by marudor** [comment]
The travis error is rx for flow 0.29 - merging.

**Pull request merged**